shell/browser/api/electron_api_base_window.cc

```diff
@@ -953,6 +953,14 @@ void BaseWindow::AddTabbedWindow(NativeWindow* window,
     args->ThrowError("AddTabbedWindow cannot be called by a window on itself.");
 }
 
+v8::Local<v8::Value> BaseWindow::GetTabbingIdentifier() {
```

Comment:
If you make the `GetTabbingIdentifier` methods return `absl::optional<std::string>` then you can remove the manual conversations between v8 value and string.

Reply:
@zcbenz gin seems to convert `absl::nullopt` into `null` when there's no value - is that something you'd prefer?